packages/apps-config/src/ui/logos/index.ts

```diff
@@ -672,7 +673,6 @@ export const namedLogos: Record<string, unknown> = {
   halongbay: nodePolkaFoundry,
   hanonycash: nodeHanonycash,
   heiko: chainParallel,
-  helixstreet: nodeHelixstreet,
```

Comment:
Not sure why this was removed here?

Reply:
Sorry. not sure why that line was removed.  I've added helixstreet back + fixed sorting issues found by the linter.   
Please check again. thank you!